lib/puppet_x/puppetlabs/powershell/powershell_manager.rb

```diff
@@ -41,11 +41,11 @@ def initialize(cmd)
         at_exit { exit }
       end
 
-      def execute(powershell_code, timeout_ms = 300 * 1000)
+      def execute(powershell_code, timeout_ms = nil, working_dir = nil)
```

Comment:
Oh I see. Thanks. Somehow I had completely missed https://github.com/puppetlabs/puppetlabs-powershell/pull/136 and MODULES-3709 and was confused why the timeout resource param didn't appear to be connected anywhere. Got it now.